packages/grant-explorer/src/features/round/ViewProjectDetails.tsx

```diff
@@ -435,17 +438,24 @@ export function ProjectStats() {
   const { chainId, roundId, applicationId } = useParams();
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   const { round } = useRoundById(chainId!, roundId!);
-  const project = round?.approvedProjects?.find(
-    (project) => project.grantApplicationId === applicationId
-  );
-  const { data } = useSWR(
-    {
-      chainId,
-      roundId,
-      projectId: project?.recipient,
-    },
-    getProjectSummary
-  );
+  const projectId = applicationId?.split("-")[0] as string
+
+  // NOTE: Consider moving this
+  function useRoundProject(roundId: string, projectId: string) {
+    // use chain id and project id from urk 
+    const client = new Client(
+      boundFetch,
+      "https://grants-stack-indexer.fly.dev",
```

Comment:
we should move the endpoint to an environment variable since it will change soon